project/MleapProject.scala

```diff
@@ -110,7 +110,9 @@ object MleapProject {
   lazy val xgboostRuntime = Project(
     id = "mleap-xgboost-runtime",
     base = file("mleap-xgboost-runtime"),
-    dependencies = Seq(runtime)
+    dependencies = Seq(
+      runtime,
+      sparkTestkit % "test")
```

Comment:
where do we use sparkTestkit?

Reply:
If I don't bring that in, I cannot use:
`import org.apache.spark.sql.mleap.TypeConverters`

 Do you think it is ok to use that in tests? (%"test" should not be included in the build)

Reply:
I think that comes from mleap-spark-base, but spark-testkit should be ok too.